content/en/getting_started/synthetics/browser_test.md

```diff
@@ -0,0 +1,76 @@
+## Create a browser test
```

Comment:
Can we have a bigger font here? So that it looks like an actual page title? Like we have in https://docs.datadoghq.com/getting_started/application/monitors for instance?

Reply:
Addressed in the last comment - editing now.